packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -474,13 +493,31 @@ function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Pr
                 />
 
                 {isMultiTouching && (
-                  <TouchPointMarker x={touchPoint.x} y={touchPoint.y} isPressing={isPressing} />
+                  <TouchPointMarker
+                    x={touchPoint.x}
+                    y={touchPoint.y}
+                    isPressing={isPressing}
+                    color="175, 175, 175"
```

Comment:
Why do you add those values here /or why did you defined default ones in the component definition. 
Also please extract magic numbers like that to constants.